src/wallet/wallet-generation.ts

```diff
@@ -29,6 +30,9 @@ async function generateFaucetWallet(
   this: RippleAPI,
   address?: string
 ): Promise<FaucetWallet | void> {
+  if(!this.isConnected())
```

Comment:
I know this would be a "new" feature, but: generally, the ideal would be to just auto-connect if we need to connect and aren't connected yet.

Perhaps beyond the scope of this particular PR

Reply:
Can we `await this.connect()` here, then?

Reply:
I think so... although: what's the harm in not connecting? The faucet API is a service provided by RippleX; it is independent of RippleAPI's rippled connection.

Reply:
The faucet waits on the balance of the account to change before returning.

So if there is not a connection, it'll time out